docs/configuration/appointment.rst

```diff
@@ -0,0 +1,51 @@
+.. _configuration_appointment:
+
+=========================
+Appointment configuration
+=========================
+
+Appointments can be made using two separate services.  JCC and Qmatic.
```

Comment:
Can you split up JCC and Qmatic in separate files, in a folder `appointment` similar to the other plugins: https://github.com/open-formulieren/open-forms/tree/master/docs/configuration

This was changed recently!